libs/datamodel/core/src/validate/common.rs

```diff
@@ -1,43 +0,0 @@
-use crate::{
-    ast,
-    common::preview_features::*,
-    diagnostics::{DatamodelError, DatamodelWarning, Diagnostics},
-};
-use itertools::Itertools;
-
-pub fn parse_and_validate_preview_features(
```

Comment:
This just moved. One module fewer. It's only used in one place, so now it's there and it is private.